x-pack/plugins/apm/public/components/shared/ManagedTable/index.tsx

```diff
@@ -60,7 +60,7 @@ function UnoptimizedManagedTable<T>(props: Props<T>) {
   const renderedItems = useMemo(() => {
     // TODO: Use _.orderBy once we upgrade to lodash 4+
     const sortedItems = sortItems
-      ? sortByOrder(items, sortField, sortDirection)
+      ? orderBy(items, sortField, sortDirection as any)
```

Comment:
You can avoid `any` by changing a line in `x-pack/plugins/apm/public/context/UrlParamsContext/types.ts`:

```diff
- sortDirection?: 'asc' | 'desc';
+ sortDirection?: string;
```

```suggestion
      ? orderBy(items, sortField, sortDirection)
```

Reply:
I tried some alternatives you suggested but I had the need to do `as 'asc' | 'desc'` as it being done in another places of that file